index.js

```diff
@@ -11,7 +11,9 @@ module.exports = {
 		const mapped = map(path, 'qunit2-parameterize.js', (content) => {
 			const newContent = [
 				";define('qunit-parameterize', ['exports', 'ember-qunit'], function(exports, emberQunit) {\n",
-				'var QUnit = window.QUnit; emberQunit.extend = QUnit.extend; QUnit = emberQunit;\n',
+				'var QUnit = window.QUnit;\n',
+				'emberQunit.extend = QUnit.extend; emberQunit.skip = QUnit.skip; emberQunit.test = QUnit.test;\n',
```

Comment:
Maybe, we no longer need to extend `emberQunit`. Did you try to use only QUnit module?

Reply:
It seems that using
```js
const newContent = [
	";define('qunit-parameterize', ['exports'], function(exports) {\n",
	'var QUnit = window.QUnit;\n',
	content,
	"exports['default'] = QUnit.cases.init.bind(QUnit.cases);\n});"
];
```

Is working fine on our project, but i don't know why in a first place we had the need to have `QUnit = emberQunit` here?

Reply:
If I recall correctly, it was a legacy thing when ember exported its own `test` function to be able to use it instead of QUnit one. I think we can remove it now.